csrc/device_lower/lower2device.h

```diff
@@ -51,16 +52,17 @@ class GpuLower : public NonCopyable {
  public:
   GpuLower() = delete;
 
+  using Pass = std::pair<
```

Comment:
nit: some comment about `Pass` would be great to have, e.g., what the string means, what the function input and output are.

Reply:
added